react/IntentIframe/styles.styl

```diff
@@ -0,0 +1,17 @@
+iframe
+    width 100%
+    height 100%
+    border 0
+
+
+.intentIframe
+    height 100vh
```

Comment:
Watch out, `100vh` on Safari/Chrome is actually longer than 100% of the viewport's height. 
Sometimes it doesn't render as you wished for. Did you test it on those browsers, preferably on mobile?

Reply:
Thanks, Is using 100% safer ?

Reply:
well yeah but a little more verbose as you need this element's parents to have their height set to `100%` too but I guess that's already the case, so it's good. :)